yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -516,59 +516,109 @@
+                       {{ if not .onlyPerf }}
+                            <th class="centered-cell" style="cursor: pointer;" onclick="sortTableBy('category', {{ .onlyPerf }})">
                             Category <span class="sort-indicator">&#8597;</span>
-                        </th>
-                        <th class="centered-cell" style="cursor: pointer;" onclick="sortTableBy('name', false)">
-                            Issue <span class="sort-indicator">&#8597;</span>
+                            </th>
+                       {{ end }}
+                        <th class="centered-cell" style="cursor: pointer;" onclick="sortTableBy('name', {{ .onlyPerf }})">
+                            {{ $issueKeyword }} <span class="sort-indicator">&#8597;</span>
                         </th>
                         <th class="centered-cell">Object/SQL Preview</th>
-                        <th class="centered-cell" style="cursor: pointer; text-align: center;" onclick="sortTableBy('impact',false)">
+                        <th class="centered-cell" style="cursor: pointer; text-align: center;" onclick="sortTableBy('impact', {{ .onlyPerf }})">
                             Impact <span class="sort-indicator">&#8597;</span>
                         </th>
                     </tr>
                 </thead>
-                {{ range $idx, $issue := $allIssuesWithoutPerfOptimzations }}
+                {{ $onlyPerf := .onlyPerf }}
+                {{ range $idx, $issue := $issues }}
+                    {{ $detailsId := "details" }}
+                    {{ $arrowId := "arrow"}}
+                    {{ if $onlyPerf }} 
+                        {{ $detailsId = "details-perf" }}
+                        {{ $arrowId = "arrow-perf"}}
+                    {{ end }}
                     <tbody class="issue-summary-detail-pair"
                         data-category="{{ $issue.Category }}"
                         data-name="{{ $issue.Name }}"
                         data-impact="{{ $issue.Impact }}">
+                        
                         <!-- Summary Row -->
-                        <tr class="summary-row" onclick="toggleDetails('details-{{ $idx }}', 'arrow-{{ $idx }}', false)">
+                        <tr class="summary-row" onclick="toggleDetails('{{ $detailsId }}-{{ $idx }}', '{{ $arrowId }}-{{ $idx }}', {{ $onlyPerf }})">
+                            {{ if not $onlyPerf }}
+                                <td class="centered-cell">
+                                    <div class="category-cell-flex">
+                                        <div class="arrow-container">
+                                            <span id="{{ $arrowId}}-{{ $idx }}">&#9654;</span>
+                                        </div>
+                                        <span style="word-break: break-word;">{{ snakeCaseToTitleCase $issue.Category }}</span>
```

Comment:
i think you can apply the if condition on this line, as remaining part of cell is same for both cases.

Reply:
We don't want to display this column altogether in `onlyPerf` case, so it will be required for whole right?